<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDlg</class>
 <widget class="QDialog" name="ExportDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Translations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QGroupBox" name="includeGroupBox">
       <property name="title">
        <string>Include</string>
       </property>
       <layout class="QVBoxLayout" name="includeLayout">
        <item>
         <widget class="QCheckBox" name="checkTransCheckBox">
          <property name="text">
           <string>Translations</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="exportGroupBox">
       <property name="title">
        <string>Export</string>
       </property>
       <layout class="QVBoxLayout" name="exportLayout">
        <item>
         <widget class="QRadioButton" name="radioAllRadioButton">
          <property name="text">
           <string>&amp;All strings</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioChangesRadioButton">
          <property name="text">
           <string>C&amp;hanges only</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioNonDialogRadioButton">
          <property name="text">
           <string>&amp;Non dialog</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioDialogRadioButton">
          <property name="text">
           <string>&amp;Dialog only</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioSampleRadioButton">
          <property name="text">
           <string>&amp;Sample</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioUnverifiedRadioButton">
          <property name="text">
           <string>&amp;Unverified dialog</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioMissingRadioButton">
          <property name="text">
           <string>&amp;Missing dialog</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioUnsentRadioButton">
          <property name="text">
           <string>Unsen&amp;t</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="languageLayout">
     <item>
      <widget class="QLabel" name="languageLabel">
       <property name="text">
        <string>Language:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="langComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filenameLayout">
     <item>
      <widget class="QLabel" name="filenameLabel">
       <property name="text">
        <string>Filename:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filenameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>&amp;Export</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>



